ecommerce/templates/checkout_interstitial.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Checkout Interstitial</title>
+</head>
+<body>
+    <h1 style="font-family: Montserrat, sans-serif">Redirecting to the payment processor...</h1>
+
+    <form id="checkout_form" method="post" action="{{ checkout_payload.url }}">
+        {% for key, value in form.items %}
+        <input style="display: none;" type="text" readonly="readonly" name="{{ key }}" value="{{ value }}" /></p>
```

Comment:
This can be a hidden input instead, also there's a trailing and unmatched `</p>`at the end of this line.